serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc

```diff
@@ -0,0 +1,32 @@
+= Chrome Extension for Serverless Workflow Editor
+
+The Chrome Extension for Serverless Workflow Editor allows you to view and edit Serverless Workflow files in github.
+
+== Installing Chrome Extension
+The chrome extension can be installed in two ways.
+
+* You can visit chrome web store with https://chrome.google.com/webstore/detail/serverless-workflow-edito/ijamhkegfogkfnmfnfkjdiadomlphfej[Chrome extension for serverless workflow editor] and click on *Add to Chrome* button.
```

Comment:
maybe use the link patter as we are doing on other guides?
common link on antora.yaml and links only used here, below the header.